tests/phpunit/ParametersFetcherTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\SnowflakeParametersExtractor\Tests;
+
+use Keboola\Db\Import\Snowflake\Connection;
+use Keboola\SnowflakeParametersExtractor\ParametersFetcher;
+use PHPUnit\Framework\TestCase;
+
+class ParametersFetcherTest extends TestCase
+{
+
+    public function testFetchParameters(): void
+    {
+        $connection = new Connection([
+            'host' => getenv('SNOWFLAKE_HOST'),
+            'user' => getenv('SNOWFLAKE_USER'),
+            'password' => getenv('SNOWFLAKE_PASSWORD'),
+        ]);
+
+        $accountParameters = (new ParametersFetcher($connection))->fetchAccountParameters();
+        $firstParameter = reset($accountParameters);
+
+        $this->assertNotEmpty($accountParameters);
```

Comment:
it's better to avoid calling static method 'assertNotEmpty' dynamically :)

Reply:
Všichni to volají nad instancí a já jsem taky tak zvyklý.
- https://github.com/aws/aws-sdk-php/blob/6c225154ac6f767c8135fc13a35d0aab1399288d/tests/ClientResolverTest.php#L92
- https://github.com/guzzle/guzzle/blob/c9615e7a872f18f63970b558f2555697f03e7803/tests/ClientTest.php#L42
- https://github.com/symfony/symfony/blob/e6302e2c920fc9c282389998dd30127bd1d48ad5/src/Symfony/Component/Config/Tests/ConfigCacheTest.php#L46

Reply:
https://keboola.slack.com/archives/C02C3GZUW/p1517392718000199